src/Introspection/__InputValue.hack

```diff
@@ -3,17 +3,28 @@ namespace Slack\GraphQL\Introspection;
 use namespace Slack\GraphQL;
 
 <<GraphQL\ObjectType('__InputValue', 'Input value introspection')>>
-interface __InputValue {
+final class __InputValue {
+
+    public function __construct(private string $name, private GraphQL\Types\InputObjectType::TField $field) {}
 
     <<GraphQL\Field('name', 'Name of the field')>>
-    public function getName(): string;
+    public function getName(): string {
+        return $this->name;
+    }
 
     <<GraphQL\Field('description', 'Description of the field')>>
-    public function getDescription(): ?string;
+    public function getDescription(): ?string {
+        return $this->field['description'] ?? null;
+    }
 
     <<GraphQL\Field('type', 'Type of the field')>>
-    public function getType(): __Type;
+    public function getType(): __Type {
+        $type = $this->field['type'];
+        return $this->field['is_nullable'] ? $type::nullableInput() : $type::nonNullable();
```

Comment:
After the change discussed above, this check shouldn't be needed, since the method call chain returned by `input_type` should already correctly be `nullableInput`/`nonNullable`